### PR TITLE
Remove engineering tool sample

### PR DESCRIPTION
**Pull request by marcschier**
* Remove engineering tool sample
* Remove signalr support required by engineering tool
* Remove telemetry processor support.

### CODE REVIEW

**Review by mregen — APPROVED** [review]
:shipit:

File: docs/services/readme.md [hunk 1]
```diff
@@ -34,8 +34,6 @@ The following Agents are part of the platform and handle event processing as wel
 
 An alternative to hosting individual containers in a cluster is the "[all in one service container](all-in-one.md)", which is a single web service that hosts all micro services and agents in its process.  
 
-A sample web application is provided in the form of the Azure Industrial IoT [Engineering Tool](engineeringtool.md), which is deployed using the [deployment script](../deploy/howto-deploy-all-in-one.md).
```
Comment (koepalex) [inline docs/services/readme.md:37]: Should we add the link to the repository were engineering tool is move to?

**Review by koepalex — APPROVED** [review]
:shipit:

**Comment by mregen** [comment]
please hold merge back until further notice to not destabilize again.

**Pull request closed without merging**